examples/hello-world/src/index.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import {IntlProvider} from 'react-intl';
+import {IntlProvider,addLocaleData} from 'react-intl';
 import App from './App';
-
+import fr from 'react-intl/locale-data/fr';
+addLocaleData(fr);
```

Comment:
Fair enough.  Maybe when someone has time we can add a multi-language demo which demonstrates multiple languages on one page.  Although the irony of wanting Hello World to be in a single language is not lost on me :)